swssconfig/swssconfig.cpp

```diff
@@ -0,0 +1,201 @@
+#include <stdlib.h>
+#include <iostream>
+#include <vector>
+#include "logger.h"
+#include <fstream>
+#include "dbconnector.h"
+#include "producertable.h"
+#include "json.hpp"
+
+using namespace std;
+using namespace swss;
+using json = nlohmann::json;
```

Comment:
It's source file (CPP) and not include, it shouldn't cause conflicts. I think it's okay.